src/components/tx-flow/flows/EnableRecovery/index.tsx

```diff
@@ -0,0 +1,96 @@
+import type { ReactElement } from 'react'
+
+import TxLayout from '@/components/tx-flow/common/TxLayout'
+import RecoveryPlus from '@/public/images/common/recovery-plus.svg'
+import useTxStepper from '../../useTxStepper'
+import { EnableRecoveryFlowReview } from './EnableRecoveryFlowReview'
+import { EnableRecoveryFlowSettings } from './EnableRecoveryFlowSettings'
+import { EnableRecoveryFlowIntro } from './EnableRecoveryFlowIntro'
+
+const DAY_SECONDS = 60 * 60 * 24
+
+export const RecoveryDelayPeriods = [
+  {
+    label: '2 days',
+    value: `${DAY_SECONDS}`,
+  },
+  {
+    label: '7 days',
+    value: `${DAY_SECONDS * 7}`,
+  },
+  {
+    label: '14 days',
+    value: `${DAY_SECONDS * 14}`,
+  },
+  {
+    label: '28 days',
+    value: `${DAY_SECONDS * 28}`,
+  },
+  {
+    label: '56 days',
+    value: `${DAY_SECONDS * 56}`,
+  },
+] as const
+
+export const RecoveryExpirationPeriods = [
+  {
+    label: 'Never',
+    value: '0',
+  },
+  ...RecoveryDelayPeriods,
+] as const
+
+export enum EnableRecoveryFlowFields {
+  guardians = 'guardians',
+  txCooldown = 'txCooldown',
+  txExpiration = 'txExpiration',
+  emailAddress = 'emailAddress',
+}
+
+export type EnableRecoveryFlowProps = {
+  [EnableRecoveryFlowFields.guardians]: Array<string>
```

Comment:
What is the reason for it being an array if the user can only select one guardian address?

Reply:
This is leftover from a design change. I've converted it to a string in 0fb37869516077a163a979e0269b1eecacddd993.